### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by danielchristopher1**
Some small changes I made while taking a look through the codebase.

### CODE REVIEW

File: QKSMS/src/main/java/com/moez/QKSMS/data/ContactHelper.java [hunk 1]
```diff
@@ -192,7 +192,7 @@ public static Bitmap getBitmap(Context context, long id) {
 
         return bitmap;
     }
-
+
```
Comment (moezbhatti) [inline QKSMS/src/main/java/com/moez/QKSMS/data/ContactHelper.java]: This one should probably be reverted, don't need spaces here

**Comment by moezbhatti** [comment]
Aside from that one small thing, looks good. If you can remove those spaces I'll go ahead with the merge!

**Comment by danielchristopher1** [comment]
I think it's good. Sorry for the irrelevant changes lol.. I wanted to start contributing to this project so I just decided to submit a tiny PR and get used to the flow of working on qksms.

**Comment by moezbhatti** [comment]
@danielchristopher1 awesome, well if you need any help understanding things on here then feel free to shoot me a message any time on slack

**Pull request merged**